README.md

```diff
@@ -120,20 +132,42 @@ _There's no way to avoid all the error (short of doing a four-dimensional realig
 
 ## Boiler plate methods section
 
-Work in progress...
+### Preprocessing
+
+The fMRI data were pre-processed and analyzed using statistical parametric mapping (SPM12 – v7487; Wellcome Center for Neuroimaging, London, UK; www.fil.ion.ucl.ac.uk/spm) running on {octave 4.{??} / matlab 20{XX} (Mathworks)}.
+
+The preprocessing of the functional images was performed in the following order: removing of dummy scans, {slice timing correction}, realignment, normalization to MNI, smoothing.
```

Comment:
(I might be wrong) on the last meeting (1st oct) we discussed about how to deal with the dummies removal/onset time and @mohmdrezk proposed to remove them before BIDS conversion (and therefore not mention them?). Is it correct?

Reply:
True. I agree with what @mohmdrezk said below.